### PR TITLE
Clean up some code related to tile drawing.

### PR DESCRIPTION
**Pull request by asvitkine**
## Change Summary & Additional Notes

Clean up some code related to tile drawing. No functional changes.

### CODE REVIEW

File: game-app/game-core/src/main/java/games/strategy/triplea/ui/panels/map/MapPanel.java [hunk 1]
```diff
@@ -852,16 +851,15 @@ public void setMouseShadowUnits(final Collection<Unit> units) {
...
 
-    final Tuple<BigDecimal, BigDecimal> movementLeft =
+    final Range<BigDecimal> movementLeft =
         getMinAndMaxMovementLeft(
             CollectionUtils.getMatches(units, Matches.unitIsBeingTransported().negate()));
     movementLeftForCurrentUnits =
-        movementLeft.getFirst()
-            + (movementLeft.getSecond().compareTo(movementLeft.getFirst()) > 0 ? "+" : "");
+        movementLeft.getMinimum()
+            + (movementLeft.getMinimum().compareTo(movementLeft.getMaximum()) > 0 ? "+" : "");
```
Comment (RoiEXLab) [inline game-app/game-core/src/main/java/games/strategy/triplea/ui/panels/map/MapPanel.java:859]: If I'm not mistaken Minimum and Maximum have to be swapped here

File: game-app/game-core/src/main/java/games/strategy/triplea/ui/panels/map/MapPanel.java [hunk 2]
```diff
@@ -701,11 +706,7 @@ public void paint(final Graphics g) {
     }
     for (final Collection<Unit> value : highlightedUnits) {
       for (final UnitCategory category : UnitSeparator.categorize(value)) {
-        final List<Unit> territoryUnitsOfSameCategory = category.getUnits();
-        if (territoryUnitsOfSameCategory.isEmpty()) {
-          continue;
-        }
-        final Rectangle r = tileManager.getUnitRect(territoryUnitsOfSameCategory, gameData);
```
Comment (RoiEXLab) [inline game-app/game-core/src/main/java/games/strategy/triplea/ui/panels/map/MapPanel.java:708]: Good change (I assume this is the only usage of the getUnitRect method)

**Review by RoiEXLab — CHANGES_REQUESTED** [review]

File: game-app/game-core/src/main/java/games/strategy/triplea/ui/panels/map/MapPanel.java (hunk 2 above)
Comment (asvitkine) [reply game-app/game-core/src/main/java/games/strategy/triplea/ui/panels/map/MapPanel.java:708]: Yep!

File: game-app/game-core/src/main/java/games/strategy/triplea/ui/panels/map/MapPanel.java (hunk 1 above)
Comment (asvitkine) [reply game-app/game-core/src/main/java/games/strategy/triplea/ui/panels/map/MapPanel.java]: Good catch, fixed!

**Review by RoiEXLab — APPROVED** [review]

**Pull request merged**